account_banking_sepa_direct_debit/README.rst

```diff
@@ -49,7 +49,7 @@ Known issues / Roadmap
 ======================
```

Comment:
Can you take the opportunity to correct the README following the newest template?